### DISCUSSION TITLE
note@enclose

### DISCUSSION THREAD

**Discussion by craigsapp** [rism-digital/verovio#3015] [Q&A] [answered]
Is there an implementation of `note@enclose` in verovio?  I thought I saw examples of such several months ago, but maybe not since I cannot see anything about it (perhaps it was discussion on the music-encoding repository).

**Comment by rettinghaus** [comment, marked as answer]
I think you are referring to [this issue](https://github.com/music-encoding/music-encoding/issues/672)? As soon as that has been cleared there will be an implementation in Verovio. 😉

**Comment by craigsapp** [comment]
Yes, that is exactly the one I was thinking about.  Only two years old 😜
